### PR TITLE
Practitioner attachment update.

### PR DESCRIPTION
**Pull request by harrymaynard-maximus**

### CODE REVIEW

File: src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                  [errorMessage]="sameMailAddrError">
+    </common-radio>
+  </div>
+
+  <ng-container *ngIf='shouldShowNewSection'>
+    <div class="form-group">
+      <common-radio formControlName="newAttachmentType"
+                    name="newAttachmentType"
+                    label="Is this attachment a Locum or temporary?"
+                    [radioLabels]='[{label:"Yes", value: true},{label: "No", value: false}]'
```
Comment (kreed0422) [inline src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.html]: common-radio default labels to yes/no.

          <common-radio name="newAttachmentTyp"' label="Is this attachment a Locum or temporary?"
                        formControlName="newAttachmentType"></common-radio>

File: src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import { Component, OnInit } from '@angular/core';
+import { RegistrationForm } from '../../models/registration-form';
+import { PRACTITIONER_REGISTRATION_PAGES } from '../../practitioner-registration-route-constants';
+import { RegistrationContainerService } from '../../services/registration-container.service';
+import { Router } from '@angular/router';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { CreatePractitionerDataService } from '../../services/create-practitioner-data.service';
+import { PRACTITIONER_ATTACHMENT } from '../../models/practitioner-attachment';
+
+interface RadioItem {
```
Comment (kreed0422) [inline src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts]: IRadioItems <= Interface for radio buttons labels, you do not need to add RadioItem.

File: src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.html (hunk 1 above)
Comment (harrymaynard-maximus) [reply src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.html]: Done.

File: src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts (hunk 2 above)
Comment (harrymaynard-maximus) [reply src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts]: Done.

**Pull request merged**